tests/python/test_percpu.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+# Copyright (c) PLUMgrid, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License")
+
+from bcc import BPF
+from time import sleep
+import unittest as ut
```

Comment:
No need to abbreviate unittest, renames like this should really be used sparingly. 'ut' only shows up twice in the file and is unlikely to grow.